python/lsst/ts/mtmount/mtmount_csc.py

```diff
@@ -843,30 +852,87 @@ async def send_command(self, command, do_lock):
             commands can run simultaneously, so specify True when practical.
             Specify False for stop commands and the camera cable wrap
             tracking command (so rotator following is not blocked).
+        timeout : `float` | `None`
+            Timeout for initial ack.
+            If None then use self.config.ack_timeout.
```

Comment:
It looks like `timeout` is always passed on to `_basic_send_command`, which then decides whether `self.config.ack_timeout` is used or not. So to me mentioning `self.config.ack_timeout` in this doc string here doesn't clarify what happens if `timeout is None`. Not sure how to improve this though, sorry.

Reply:
If timeout is None then timeout=None is passed to _basic_send_command, which then uses `self.config.ack_timeout`. That is what I tried to say in the doc string. Do you read it in some other way?

Perhaps you are worried about the way in which None is turned into self.config.ack_timeout (in _basic_send_command instead of in this public method)? If so, I consider that an implementation detail. I would be very reluctant to try to explain that detail in this doc string, since I don't see how it affects the user and it might change.